guide/command-handling/README.md

```diff
@@ -99,7 +99,38 @@ for (const file of commandFiles) {
 }
 ```
 
-As for setting up your files, that's it for now. In the next chapter, you'll learn how to make your commands execute dynamically!
+## Dynamically executing commands
+
+With your `client.commands` Collection setup, you can use it to retrieve and execute your commands! Inside your `message` event, delete your `if`/`else if` chain of commands and replace it with this:
+
+```js {7-14}
+client.on('message', message => {
+	if (!message.content.startsWith(prefix) || message.author.bot) return;
+
+	const args = message.content.slice(prefix.length).trim().split(/ +/);
+	const command = args.shift().toLowerCase();
+
+	if (!client.commands.has(command)) return;
+
+	try {
+		client.commands.get(command).execute(message, args);
```

Comment:
Not sure what concepts we want to cover here. Should the function be async and await the `.execute()`? A rejection in an async execute function won't be caught by this handler.

Reply:
I noticed this before the PR but didn't include it in this one because it'd require a few more edits in other files. Another PR can be made to make the handler use async/await